src/BloomExe/ToPalaso/AudioRecorder.cs

```diff
@@ -0,0 +1,540 @@
+﻿using System;
```

Comment:
I need to ask you (probably verbally in standup) about what's up with this, since it appears to be all linux.